e2e/support/helpers/e2e-custom-column-helpers.ts

```diff
@@ -52,3 +57,150 @@ export function checkExpressionEditorHelperPopoverPosition() {
     });
   });
 }
+
+export const CustomExpressionEditor = {
+  get() {
+    return cy.findByTestId("custom-expression-query-editor");
+  },
+  type(
+    text: string,
+    {
+      allowFastSet = false,
+      focus = true,
+    }: {
+      focus?: boolean;
+      allowFastSet?: boolean;
+    } = {},
+  ) {
+    if (focus) {
+      CustomExpressionEditor.focus();
+    }
+
+    if (allowFastSet) {
+      // Enter the formula in one go
+      // HACK: we do invoke("text") instead of type() because type() does not work on
+      // CodeMirror elements in Cypress. realType() would work but some of the formulas
+      // contain special characters that are not supported by realType().
+      CustomExpressionEditor.get().findByRole("textbox").invoke("text", text);
```

Comment:
you can achieve the same thing by `.type("text", {timeout: 0})` the advantage of this is that you’ll get slightly smaller chance of flakiness, because `.type()` will check if the element is not disabled, etc.

Reply:
nevermind, I misread the comment